contracts/WildcardSteward_v3.sol

```diff
@@ -487,15 +513,19 @@ contract WildcardSteward_v3 is Initializable {
         address tokenPatron,
         uint256 timeSinceLastMint
     ) internal {
-        mintManager.tokenMint(
-            tokenPatron,
-            timeSinceLastMint,
-            totalPatronTokenGenerationRate[tokenPatron]
-        );
-        emit CollectLoyalty(
-            tokenPatron,
-            timeSinceLastMint.mul(totalPatronTokenGenerationRate[tokenPatron])
-        );
+        if (timeSinceLastMint != 0) {
+            mintManager.tokenMint(
+                tokenPatron,
+                timeSinceLastMint,
+                totalPatronTokenGenerationRate[tokenPatron]
+            );
+            emit CollectLoyalty(
+                tokenPatron,
+                timeSinceLastMint.mul(
+                    totalPatronTokenGenerationRate[tokenPatron]
+                )
+            );
+        }
```

Comment:
This prevents needlessly minting zero tokens for users. This happens when a user has multiple tokens and they default. When their 2nd, 3rd etc... tokens foreclose, they previously would've had zero tokens minted for them.